views/templates/front/paymentOptions/paypal.tpl

```diff
@@ -17,7 +17,7 @@
  * International Registered Trademark & Property of PrestaShop SA
  *}
 
-<link rel="preload" href="https://www.paypal.com/sdk/js?components=hosted-fields,buttons&amp;client-id={$paypalClientId|escape:'htmlall':'UTF-8'}&amp;merchant-id={$merchantId|escape:'htmlall':'UTF-8'}&amp;intent={$intent|escape:'htmlall':'UTF-8'}&amp;currency={$currencyIsoCode|escape:'htmlall':'UTF-8'}&amp;locale={$locale|escape:'htmlall':'UTF-8'}" as="script">
+<link rel="preload" href="{$paypalSdkLink|escape:'htmlall':'UTF-8'}" as="script">
```

Comment:
Why did you use `escape:'htmlall':'UTF-8'` here and `nofilter` bellow ?

Reply:
Because without url is encoded like `https://www.paypal.com/sdk/js?components=buttons&amp;locale=fr_FR&amp;currency=EUR&amp;intent=capture&amp;disable-funding=card` and a javascript error 👍